.github/cspell.json

```diff
@@ -5,10 +5,19 @@
         ".git/"
     ],
     "ignorewords": [],
+    "ignoreRegExpList": [
+        "/```[\\s\\S]*?```/g",
+        "/`[\\s\\S]*?`/g"
+    ],
     "import": [],
     "dictionarydefinitions": [],
     "dictionaries": [],
     "words": [
-        "ccnp"
```

Comment:
Please use alphabet sequence to sort dictionary words.

Reply:
Add the following words if there's not.
GRPC
PaaS
SaaS
RTMR
grpcurl

Reply:
Done.  BTW, words that are used inside code blocks will be ignored for cspell scan.